### PR TITLE
Docs: Created korean manual pages.

### PR DESCRIPTION
**Pull request by hareha**
Hi, I'm a developer from South Korea.

I've checked former PR about korean translations, 

https://github.com/mrdoob/three.js/pull/20292#issue-482119452
https://github.com/mrdoob/three.js/issues/20683#issue-742794197

but I didn't find anything updated afterwards.
So I translated manual part to korean(../docs/manual/kr).

based on english text, also referred to Chinese translated docs.

### CODE REVIEW

**Comment by Mugen87** [comment]
Notice that translated pages will only be visible in the side navigation if you edit `doc/list.json`. You need similar entries like for the existing languages.

File: docs/manual/kr/buildTools/Testing-with-NPM.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (Mugen87) [inline docs/manual/kr/buildTools/Testing-with-NPM.html:2]: This should be change to the actual language of the page:
```html
<html lang="kr">
```

Comment (hareha) [reply docs/manual/kr/buildTools/Testing-with-NPM.html]: updated all <html lang="kr"> and doc/list.json files.

File: docs/list.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const list = {
+{
```
Comment (Mugen87) [inline docs/list.json:1]: Please undo these changes to `list.json`. It seems not right what you have done in this file.

**Comment by Mugen87** [comment]
It seems to me you are not working on the latest `dev` branch. `list.js` does not exist anymore. It's probably best if you rebase.

**Comment by Mugen87** [comment]
Closing in favor of #20872.
